builtin/files/stack-templates/etcd.json.tmpl

```diff
@@ -367,15 +367,25 @@
       },
       {{end}}
       "UpdatePolicy" : {
+        "AutoScalingScheduledAction" : {
+          "IgnoreUnmodifiedGroupSizeProperties" : true
+        },
         "AutoScalingRollingUpdate" : {
           "MinInstancesInService" : "0",
           "MaxBatchSize" : "1",
           {{if $.WaitSignal.Enabled}}
           "WaitOnResourceSignals" : "true",
-          "PauseTime": "{{$.Controller.CreateTimeout}}"
+          "PauseTime": "{{$.Controller.CreateTimeout}}",
           {{else}}
-          "PauseTime": "PT2M"
+          "PauseTime": "PT2M",
```

Comment:
This change feels somewhat significant, should be configurable or have you found 2m to be enough?